### DISCUSSION TITLE
[Feature Request 1.16.5] Scrollable Filter Lists

### DISCUSSION THREAD

**Discussion by p1ut0nium-git** [P3pp3rF1y/SophisticatedBackpacks#322] [Feature / Idea Archive]
I'd like the filter lists for backpack upgrades to be longer, and have a scroll bar to see all slots. Perhaps the list adds new rows when it is full?

For example: the Automatic Feeder upgrade...

I have lots of food and crop mods installed, but run out of slots very quickly when trying to avoid eating certain foods.

**Comment by P3pp3rF1y** [comment]
to be honest I don't like unlimited number of filter slots, it's one of the things that should be used for balance as even lower tier upgrades have fewer slots than higher tier ones. And in terms of food, it's likely that you can block those foods based on tags or just plain not put them in the backpack where you have feeding upgrade.
